src/pages/Home.tsx

```diff
@@ -58,9 +56,13 @@ export const Home = () => {
                               logical reasoning and proof techniques to solve
                               problems efficiently.
                             </Typography>
-                            <Button variant="outlined" size="medium">
+                            <Typography
+                              variant="button"
+                              color="primary"
+                              style={{ marginTop: '10px', display: 'block' }}

```

Comment:
use `sx` instead of style

Reply:
Removed style attribute and changed it to sx and applied the styling.